src/Helper/Cron.php

```diff
@@ -82,14 +82,12 @@ public function setShippingCompleted()
         /** @var Collection $orderCollection */
         $orderCollection = $this->shopgateOrderFactory->create()->getCollection();
         $orderCollection->filterByUnsynchronizedOrders();
+        $orderCollection->setPageSize(100);
```

Comment:
That does not do anything intended without setCurPage(1)
However why you didn't you just use limit(100) instead?

Reply:
Alex confirmed this to be working, as cur_page defaults to 1